config.mak.uname

```diff
@@ -623,6 +623,12 @@ ifeq ($(uname_S),NONSTOP_KERNEL)
 	SHELL_PATH = /usr/coreutils/bin/bash
 endif
 ifeq ($(uname_S),MINGW)
+	ifneq (,$(wildcard ../THIS_IS_MSYSGIT))
+		$(error "Building with msysGit is no longer supported")
+	endif
+	ifeq ($(shell expr "$(uname_R)" : '1\.'),2)
+		$(error "Building with MSys is no longer supported")
+	endif
```

Comment:
Since the `uname_R` condition is true also for msysGit, we do not actually need an extra `THIS_IS_MSYSGIT` clause.